### PR TITLE
fix: smartDateVerbose at seconds

### PR DESCRIPTION
**Pull request by datability-io**
🐛 Bug Fix

### CODE REVIEW

File: packages/superset-ui-time-format/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@superset-ui/time-format",
-  "version": "0.11.0",
+  "version": "0.11.1",
```
Comment (kristw) [inline packages/superset-ui-time-format/package.json]: The version number is managed automatically by `lerna`. Please do not modify the version number manually.

Comment (datability-io) [reply packages/superset-ui-time-format/package.json]: @kristw I'm reverting this line.  Do I need to do anything to increment the version number at all?  Thanks.

Comment (xtinec) [reply packages/superset-ui-time-format/package.json]: I can `yarn release` after the change is merged to up the version number, Thomas.

Comment (datability-io) [reply packages/superset-ui-time-format/package.json]: @xtinec sounds good

**Pull request merged**